tests/notebooks/e2e-wine/requirements.in

```diff
@@ -0,0 +1,9 @@
+boto3
+kfp==1.8.22
```

Comment:
Could we please add a comment on why we pin these packages? This will help us when upgrading and/or changing the contents of the `.in` files.

Reply:
Most of them are there because through experimentation it seemed as though they were the only versions that worked for our setup. This mostly refers to the super specific pins found in the e2e example (which are due to the MLflow version deployed). For KFP I can just say that until we fully move to 2.0+, things will break with the newer clients.

Reply:
Yes, I remember our conversations in other PRs, I wonder if we could reflect that information here in the file(s).

Reply:
I'll add some comments.